low-level/framework/utils/webservices.py

```diff
@@ -47,6 +48,7 @@ class WebServices(object):
     def __init__(self):
         super(WebServices, self).__init__()
 
+        init_logging("sspl", SSPL_LOG_PATH)
```

Comment:
why separate init here again?

Reply:
reply from @gargpalak048, 
```smalltalk
Note:- we need to initialize logger obj in webservices.py file as well because realstor login api is called even before sspl_ll_d script.
```

Reply:
How? sspl-ll_d has main() which starts the sspl daemon. You mean webservice class used before logging init? Then that init should simply happen first

Reply:
When [RealStorEnclosure](https://github.com/Seagate/cortx-monitor/blob/main/low-level/framework/platforms/realstor/realstor_enclosure.py) is imported, its `init` method is called in which we have a `login()` function which performs login through `realstor` API. So even before SSPL starts, we have a call to `realstor` API that uses functions from webservice class.